src/Console/Command.php

```diff
@@ -104,6 +106,26 @@ public function getName(): string
         return $this->name;
     }
 
+    /**
+     * Proxy for static calls.
+     *
+     * @param string $name Method name.
+     * @param array $arguments Method arguments.
+     * @return mixed
+     */
+    public static function __callStatic($name, $arguments)
```

Comment:
Why use the magic method instead of just having a `Command::defaultName()` method? You can throw an exception is the method is called directly on `Command` class instead of a child class.

Reply:
Or provide a default implementation on the base class.

Reply:
is `getDefaultName()` good name ?

Reply:
We could just use `defautName()` as there won't be any corresponding setter.